.circle/install-and-run-mongodb.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# Script which installs versions of MongoDB specified using an environment variable
+
+if [ ! "${MONGODB_VERSION}" ]; then
+    echo "MONGODB_VERSION environment variable not set"
```

Comment:
Actually, that's a bit hard since it's not a static list - it will work with any version which is available on MongoDB mirrors...